mlir/test/Dialect/MIOpen/translate_cflags_bwd.mlir

```diff
@@ -86,7 +86,7 @@ func @all_params(%filter : memref<?x?xf32>, %input : memref<?x?xf32>, %output :
 // CHECK: -DCK_PARAM_TUNABLE_GEMM_B_BLOCK_COPY_DST_DATA_PER_WRITE_GEMM_N=4
 // CHECK: -DCK_PARAM_TUNABLE_GEMM_B_BLOCK_COPY_SRC_DATA_PER_READ_GEMM=4
 // CHECK: -DCK_PARAM_TUNABLE_GEMM_C_THREAD_COPY_DST_DATA_PER_WRITE_GEMM_N1=1
-// CHECK: -DCK_PARAM_TUNABLE_GEMM_K_PER_BLOCK=8
```

Comment:
the value is changed because we introduce more tuning parameters from MIOpen, correct?